Python/PyPlasma.h

```diff
@@ -54,42 +53,8 @@ struct pySequenceFastRef
     PyObject* get(Py_ssize_t idx) { return PySequence_Fast_GET_ITEM(fObj, idx); }
 };
 
-// Python 3.x does things differently...  This should help to keep things
-// under control with both 2.x and 3.0 somewhat seamlessly.
-#if (PY_MAJOR_VERSION >= 3)
-    // Int -> Long
-    #define PyAnyInt_Check(ob) PyLong_Check(ob)
-    #define PyInt_FromLong PyLong_FromLong
-
-    // String -> Unicode
-    #define PyString_FromString(str) PyUnicode_DecodeUTF8((str), strlen((str)), nullptr)
-    #define PyString_FromStringAndSize(str, len) PyUnicode_DecodeUTF8((str), (len), nullptr)
-    #define PyAnyString_FromSTString PyUnicode_FromSTString
-
-    // Py_TPFLAGS_CHECKTYPES is no longer used in Py3k
-    #define Py_TPFLAGS_CHECKTYPES 0
-#elif (PY_MAJOR_VERSION == 2) && (PY_MINOR_VERSION >= 6)
-    #undef PyBytes_Check
-    #undef PyBytes_FromStringAndSize
-    #undef PyBytes_AsStringAndSize
-    #undef PyBytes_AsString
-    #undef PyBytes_AS_STRING
-
-    #define PyAnyInt_Check(ob) (PyInt_Check(ob) || PyLong_Check(ob))
-
-    #define PyAnyString_FromSTString PyString_FromSTString
-    #define PyBytes_Check(ob) PyString_Check(ob)
-    #define PyBytes_FromStringAndSize(v, len) PyString_FromStringAndSize(v, len)
-    #define PyBytes_AsStringAndSize(obj, buf, len) PyString_AsStringAndSize(obj, buf, len)
-    #define PyBytes_AsString PyString_AsString
-    #define PyBytes_AS_STRING PyString_AS_STRING
-#else
-    #error Your Python version is too old.  Only 2.6 and later are supported
-#endif
-
-// The type of hashfunc's return value changed in Python 3.2
-#if (PY_MAJOR_VERSION < 3) || ((PY_MAJOR_VERSION == 3) && (PY_MINOR_VERSION < 2))
-    typedef long Py_hash_t;
+#if (PY_MAJOR_VERSION < 3) || ((PY_MAJOR_VERSION == 3) && (PY_MINOR_VERSION < 3))
```

Comment:
The condition doesn't match the message - it should probably be
```suggestion
#if (PY_MAJOR_VERSION < 3) || ((PY_MAJOR_VERSION == 3) && (PY_MINOR_VERSION < 2))
```